docs/developers/documentation/index.md

```diff
@@ -196,7 +237,7 @@ development installation of napari installed. [Examples](gallery)
 are automatically executed when the documentation is built and code problems can
 also be caught when previewing the built documentation. If your documentation
 change does not include code, you only need the napari docs dependencies
-installed.
+installed (in which case you could also preview your document with the [VScode Myst extension](https://marketplace.visualstudio.com/items?itemName=ExecutableBookProject.myst-highlight) or [Myst live preview](https://myst-parser.readthedocs.io/en/latest/live-preview.html#)).
```

Comment:
That's fair! I find it very useful to use the VScode Myst extension to live preview what I'm writing, with full understanding that I'm not looking at exactly what will be built due to some autogeneration. I understand that could be niche though. And I included the live preview link since that is similar to the Myst extension and I didn't want to vendor lock 😂 

I think there are two options then, open to either!
❌ Let's just remove any mention of Myst previews because the autogeneration differences will cause more confusion that the benefit of very quickly seeing document changes as you write.
✔️ Take that information out of brackets and expand it with explanation. Something like: "To see the markdown document structure and content change as you write it, you can use Myst markdown preview tools (such as Vscode and live preview). However, this preview page will have some differences to the final built html page, such as missing some autogenerated images and formatting.